### PR TITLE
Fix client-side app deploy command

### PR DESCRIPTION
**Pull request by jsilvestre**

### CODE REVIEW

File: src/main.coffee [hunk 1]
```diff
@@ -108,15 +108,10 @@ program
 .action (port, slug) ->
     port = 9250 unless port?
     # Recover manifest
-    projectManager.recoverManifest port, (err, app) ->
+    options = {slug, port}
```
Comment (kosssi) [inline src/main.coffee]: it's the same as `options = {slug: slug, port: port}` ?

**Comment by m4dz** [comment]
Thanks for fixing it @jsilvestre!

File: src/main.coffee (hunk 1 above)
Comment (m4dz) [inline src/main.coffee]: @kosssi yep, can be mixed with explicit declarations too:
```coffee
{foo, bar: 'baz'}
```
outputs:
```js
{
  foo: foo,
  bar: 'baz'
}
```

Comment (kosssi) [inline src/main.coffee]: ok thanks :+1:

**Pull request merged**